src/averageAges.js

```diff
@@ -15,6 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  // const centuryOfPerson = people.map(person => Math.ceil(person.died / 100));
+
+  const malePerson = people.filter(person => person.sex === 'm');
+
+  let personAge = 0;
+  const filtredPersonByCentury = malePerson.filter(
+    person => Math.ceil(person.died / 100) === century);
+
+  const centuryOrNot = century ? filtredPersonByCentury : malePerson;
+
+  personAge = centuryOrNot.map(person => person.died - person.born);
+
+  const sumAge = personAge.reduce((sum, age) => sum + age, 0);
+
+  const averageAge = sumAge / personAge.length;
+
+  return +averageAge.toFixed(2);
```

Comment:
Create a method to calculate the average age. You repeat the same logic not once